### PR TITLE
DOC/RLS: start changelog for 0.8.0 release

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
(at some point we should maybe discuss how to handle those changelogs in the future, but for now I am fine with writing it "manually" before the release)

**Comment by martinfleis** [comment]
> (at some point we should maybe discuss how to handle those changelogs in the future, but for now I am fine with writing it "manually" before the release)

A part of the planned PR template could be a request to write a note to changelog about that PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,96 @@
...
+- IO enhancements:
+  - New `GeoDataFrame.to_postgis()` method to write to PostGIS database (#1248).
+  - New Apache Parquet and Feather file format support (#1180, #1435)
+  - Allow appending to files with `GeoDataFrame.to_file` (#1229).
+  - Add support for the `ignore_geometry` keyword in `read_file` to only read
+    the attribute data. If set to True, a pandas DataFrame without geometry is
+    returned (#1383).
+  - `geopandas.read_file` now supports reading from file-like objects (#1329).
+  - `GeoDataFrame.to_file` now supports specifying the CRS to write to the file
+  (#802). By default is still uses the CRS of the GeoDataFrame.
```
Comment (martinfleis) [inline CHANGELOG.md:25]:
```suggestion
  (#802). By default it still uses the CRS of the GeoDataFrame.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,96 @@
...
+  returned by the `sindex` attribute is deprecated and will be removed in the
+  future (#1440).
+
+Bug fixes:
+
+- Fix the `total_bounds()` method to ignore missing and empty geometries (#1312).
+- Fix `geopandas.clip` when masking with non-overlapping area resulting in an
+  empty GeoDataFrame (#1309, #1365).
+- Fix error in `geopandas.sjoin` when joining on an empty geometry column (#1318).
+- CRS related fixes: `pandas.concat` preserves CRS when concatting GeoSeries
```
Comment (martinfleis) [inline CHANGELOG.md:82]:
```suggestion
- CRS related fixes: `pandas.concat` preserves CRS when concatenating GeoSeries
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,96 @@
 Changelog
 =========
 
+Version 0.8.0 (June 24, 2020)
+-----------------------------
+
+**Experimental**: optional use of PyGEOS to speed up spatial operations (#1155).
+PyGEOS is a faster alternative for Shapely (being contributed back to a future
+version of Shapely), and is used in element-wise spatial operations and for
+spatial index in eg `sjoin` (#1343, #1401, #1421, #1427, #1428). See the
```
Comment (martinfleis) [inline CHANGELOG.md:10]:
```suggestion
spatial index in e.g. `sjoin` (#1343, #1401, #1421, #1427, #1428). See the
```

**Comment by martinfleis** [comment]
Do you want to mention #1457 in Changelog?

**Comment by jorisvandenbossche** [comment]
Thanks for the proofread! 

> Do you want to mention #1457 in Changelog?

Yep, good point, added a short sentence.

**Comment by jorisvandenbossche** [comment]
> We really have to start writing these notes with each PR to avoid the need to write it all at once.

It's not a huge problem though, also asking for it on each PR requires effort for this on each PR (just saying that both have pros/cons. But certainly given our releases get bigger, I am certainly fine with moving towards adding notes when doing PRs)

**Pull request merged**